modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/jaxrs/application/FeatureFlagApplication.java

```diff
@@ -80,6 +80,32 @@ public Set<Object> getSingletons() {
 		return Collections.singleton(this);
 	}
 
+	@Path("/is-enabled")
+	@POST
+	public Response isEnabled(
+		@Context HttpServletRequest httpServletRequest,
+		@Context HttpServletResponse httpServletResponse,
+		@FormParam("companyId") long companyId, @FormParam("key") String key) {
+
+		FeatureFlagsBag featureFlagsBag =
+			_featureFlagsBagProvider.getOrCreateFeatureFlagsBag(companyId);
+
+		FeatureFlag featureFlag = featureFlagsBag.getFeatureFlag(key);
+
+		return Response.ok(
+			HashMapBuilder.<String, Object>put(
+				"dependentFeatureFlags",
+				TransformUtil.transform(
+					_getDependentFeatureFlags(featureFlagsBag, key),
```

Comment:
does this return the transitive dependencies? (I'm not even sure they can exist, just confirming we're all ok with the model)

Reply:
Yep, that's the theory. That code was introduced to refresh the FF configuration screen after a modification without needing to reload the page.

I've been unable to check it myself, though. :grimacing: